src/app/examples/crud/books.model.ts

```diff
@@ -1,12 +1,6 @@
-import { EntityState } from '@ngrx/entity';
-
 export interface Book {
   id: string;
   title: string;
   author: string;
   description: string;
 }
-
-export interface BookState extends EntityState<Book> {
```

Comment:
Having the BookState is cleaner instead of using `EntityState<Book>`.
Also while doing this it's easier if we want to add an extra property later.

```
export interface BookState extends EntityState<Book> {}
```

Reply:
[This one](https://github.com/tomastrajan/angular-ngrx-material-starter/pull/355/commits/7413986b0b62975f9ac6ff1dfd0ec3abd3ca2711) should be fixed.